### PR TITLE
adding swagger properties for new va profile attributes

### PR DESCRIPTION
**Pull request by ericbuckley**
Adding swagger properties for the two (`va_patient` and `mhv_account_state`) new va profile attributes.

refs https://github.com/department-of-veterans-affairs/vets-api/pull/4299

### CODE REVIEW

File: app/swagger/requests/user.rb [hunk 1]
```diff
@@ -99,6 +99,8 @@ class User
                       key :type, :string
                     end
                   end
+                  property :va_patient, type: :boolean
+                  property :mhv_account_state, type: :string, enum: %w[OK DEACTIVATED MULTIPLE NONE], example: 'OK'
```
Comment (omgitsbillryan) [inline app/swagger/requests/user.rb:103]: Thoughts on adding a `description:`? `MULTIPLE` & `NONE` in particular don't seem immediately obvious to me what they mean.

File: app/swagger/requests/user.rb [hunk 2]
```diff
@@ -99,6 +99,15 @@ class User
                       key :type, :string
                     end
                   end
+                  property :va_patient, type: :boolean
+                  property :mhv_account_state,
+                           type: :string,
+                           enum: %w[OK DEACTIVATED MULTIPLE NONE],
+                           example: 'OK',
+                           description: 'DEACTIVATED: user has MHV ids that are not active; '\
```
Comment (ericbuckley) [inline app/swagger/requests/user.rb:107]: @patrickvinograd is this swagger description accurate?

Comment (patrickvinograd) [inline app/swagger/requests/user.rb:107]:
```suggestion
                           description: 'DEACTIVATED: user has at least one MHV id that is not active; '\
```

Comment (patrickvinograd) [reply app/swagger/requests/user.rb:107]: Minor phrasing update, otherwise, descriptions look accuate

**Pull request merged**